### ISSUE TITLE
GL_CLAMP_TO_BORDER undeclared issue

### ISSUE THREAD

**Issue by 6alileo** [libretro/RetroArch#8563]
## Description

Compile fails on master branch
Compile succeeds on base release 1.7.6

### Expected behavior

To compile successfully on master branch

### Actual behavior

https://pastebin.com/raw/DjDAwjff

End of log snippet

```
CC gfx/drivers_context/sdl_gl_ctx.c
CC gfx/drivers/gl_core.c
CC gfx/drivers_font/gl_core_raster_font.c
gfx/drivers/gl_core.c: In function ‘gl_core_wrap_type_to_enum’:
gfx/drivers/gl_core.c:2011:17: error: ‘GL_CLAMP_TO_BORDER’ undeclared (first use in this function); did you mean ‘GL_CLAMP_TO_BORDER_NV’?
          return GL_CLAMP_TO_BORDER;
                 ^~~~~~~~~~~~~~~~~~
                 GL_CLAMP_TO_BORDER_NV
gfx/drivers/gl_core.c:2011:17: note: each undeclared identifier is reported only once for each function it appears in
CXX gfx/drivers_shader/shader_gl_core.cpp
Makefile:199: recipe for target 'obj-unix/release/gfx/drivers/gl_core.o' failed
make: *** [obj-unix/release/gfx/drivers/gl_core.o] Error 1
make: *** Waiting for unfinished jobs....
~
Could not successfully build retroarch-dev - RetroArch (master branch) - CAUTION - may cause issues (/home/pigaming/RetroArena-Setup/tmp/build/retroarch-dev/retroarch not found).

Log ended at: Mon Apr  8 17:04:56 UTC 2019
Total running time: 0 hours, 2 mins, 23 secs
```

### Environment information

Odroid-N2
Ubuntu 18.04.2 LTS (GNU/Linux 4.9.162+ aarch64)

**Comment by bparker06** [comment]
Confirmed bug, GL_CLAMP_TO_BORDER is only defined in the headers for GLES 3.2 when we're supposed to be targeting 3.0.

**Comment by bparker06** [comment]
Should be fixed in 3346cd23c03b539e7653ddead89adb1ed495314a

**Comment by 6alileo** [comment]
It's fixed but now I have new issues? Do you want me to close and open a new issue?

https://pastebin.com/raw/YpQUKuSp

```
CC cheevos/var.c
CC cheevos/cond.c
CXX deps/discord-rpc/src/discord_rpc.cpp
CXX deps/discord-rpc/src/rpc_connection.cpp
In file included from deps/discord-rpc/src/rpc_connection.h:4:0,
                 from deps/discord-rpc/src/rpc_connection.cpp:1:
deps/discord-rpc/src/serialization.h:6:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(push)
 
deps/discord-rpc/src/serialization.h:8:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4061) // enum is not explicitly handled by a case label
 
deps/discord-rpc/src/serialization.h:9:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4365) // signed/unsigned mismatch
 
deps/discord-rpc/src/serialization.h:10:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4464) // relative include path contains
 
deps/discord-rpc/src/serialization.h:11:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4668) // is not defined as a preprocessor macro
 
deps/discord-rpc/src/serialization.h:12:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 6313) // Incorrect operator
 
CXX deps/discord-rpc/src/serialization.cpp
In file included from deps/discord-rpc/src/serialization.cpp:1:0:
deps/discord-rpc/src/serialization.h:6:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(push)
 
deps/discord-rpc/src/serialization.h:8:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4061) // enum is not explicitly handled by a case label
 
deps/discord-rpc/src/serialization.h:9:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4365) // signed/unsigned mismatch
 
deps/discord-rpc/src/serialization.h:10:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4464) // relative include path contains
 
deps/discord-rpc/src/serialization.h:11:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4668) // is not defined as a preprocessor macro
 
deps/discord-rpc/src/serialization.h:12:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 6313) // Incorrect operator
 
In file included from deps/discord-rpc/src/rpc_connection.h:4:0,
                 from deps/discord-rpc/src/discord_rpc.cpp:6:
deps/discord-rpc/src/serialization.h:6:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(push)
 
deps/discord-rpc/src/serialization.h:8:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4061) // enum is not explicitly handled by a case label
 
deps/discord-rpc/src/serialization.h:9:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4365) // signed/unsigned mismatch
 
deps/discord-rpc/src/serialization.h:10:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4464) // relative include path contains
 
deps/discord-rpc/src/serialization.h:11:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 4668) // is not defined as a preprocessor macro
 
deps/discord-rpc/src/serialization.h:12:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(disable : 6313) // Incorrect operator
 
In file included from deps/discord-rpc/src/rpc_connection.h:4:0,
                 from deps/discord-rpc/src/rpc_connection.cpp:1:
deps/discord-rpc/src/serialization.h:20:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(pop)
 
In file included from deps/discord-rpc/src/rpc_connection.h:4:0,
                 from deps/discord-rpc/src/discord_rpc.cpp:6:
deps/discord-rpc/src/serialization.h:20:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(pop)
 
In file included from deps/discord-rpc/src/serialization.cpp:1:0:
deps/discord-rpc/src/serialization.h:20:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(pop)
 
CC discord/discord.c
CXX deps/discord-rpc/src/discord_register_linux.cpp
CXX deps/discord-rpc/src/connection_unix.cpp
CC input/input_remote.c
CC cores/libretro-net-retropad/net_retropad_core.c
CC deps/miniupnpc/igd_desc_parse.c
CC deps/miniupnpc/upnpreplyparse.c
CC deps/miniupnpc/upnpcommands.c
CC deps/miniupnpc/upnperrors.c
CC deps/miniupnpc/connecthostport.c
CC deps/miniupnpc/portlistingparse.c
CC deps/miniupnpc/receivedata.c
CC deps/miniupnpc/upnpdev.c
CC deps/miniupnpc/minissdpc.c
CC deps/miniupnpc/miniwget.c
CC deps/miniupnpc/miniupnpc.c
CC deps/miniupnpc/minixml.c
CC deps/miniupnpc/minisoap.c
CC record/drivers/record_ffmpeg.c
CC cores/libretro-ffmpeg/ffmpeg_core.c
CC tasks/task_decompress.c
CC led/drivers/led_rpi.c
LD retroarch
obj-unix/release/gfx/drivers/gl1.o: In function `draw_tex':
gl1.c:(.text+0xae8): undefined reference to `glBegin'
gl1.c:(.text+0xb0c): undefined reference to `glTexCoord2f'
gl1.c:(.text+0xb18): undefined reference to `glVertex2f'
gl1.c:(.text+0xb24): undefined reference to `glTexCoord2f'
gl1.c:(.text+0xb30): undefined reference to `glVertex2f'
gl1.c:(.text+0xb3c): undefined reference to `glTexCoord2f'
gl1.c:(.text+0xb48): undefined reference to `glVertex2f'
gl1.c:(.text+0xb54): undefined reference to `glTexCoord2f'
gl1.c:(.text+0xb60): undefined reference to `glVertex2f'
gl1.c:(.text+0xb64): undefined reference to `glEnd'
collect2: error: ld returned 1 exit status
Makefile:195: recipe for target 'retroarch' failed
make: *** [retroarch] Error 1
~
Could not successfully build retroarch-dev - RetroArch (master branch) - CAUTION - may cause issues (/home/pigaming/RetroArena-Setup/tmp/build/retroarch-dev/retroarch not found).

Log ended at: Mon Apr  8 21:14:42 UTC 2019
Total running time: 0 hours, 5 mins, 56 secs
```

**Comment by bparker06** [comment]
Yes please open a new issue... but for that one, you probably just need `--disable-gl` added to your configure line since it's not applicable to GLES devices.

**Comment by orbea** [comment]
The qb scripts need a bit of work with opengl, glcore and gl1, but its kind of tricky...
